modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/impl/StandaloneMetaStorageManager.java

```diff
@@ -0,0 +1,135 @@
+import org.apache.ignite.internal.metastorage.server.KeyValueStorage;
+import org.apache.ignite.internal.metastorage.server.SimpleInMemoryKeyValueStorage;
+import org.apache.ignite.internal.metastorage.server.raft.MetaStorageListener;
+import org.apache.ignite.internal.raft.Command;
+import org.apache.ignite.internal.raft.RaftManager;
+import org.apache.ignite.internal.raft.ReadCommand;
+import org.apache.ignite.internal.raft.WriteCommand;
+import org.apache.ignite.internal.raft.service.CommandClosure;
+import org.apache.ignite.internal.raft.service.RaftGroupService;
+import org.apache.ignite.internal.util.IgniteSpinBusyLock;
+import org.apache.ignite.internal.vault.VaultManager;
+import org.apache.ignite.network.ClusterNode;
+import org.apache.ignite.network.ClusterService;
+import org.apache.ignite.network.NetworkAddress;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.TestOnly;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+
+/**
+ * MetaStorageManager dummy implementation.
+ */
+@TestOnly
+public class StandaloneMetaStorageManager extends MetaStorageManagerImpl {
+    private static final String TEST_NODE_NAME = "test";
+
+    /**
+     * Creates standalone MetaStorage manager for provided VaultManager.
+     */
+    public static StandaloneMetaStorageManager create(VaultManager vaultMgr) {
+        return create(vaultMgr, new SimpleInMemoryKeyValueStorage(TEST_NODE_NAME));
+    }
+
+    /**
+     * Creates standalone MetaStorage manager for provided VaultManager.
+     */
+    public static StandaloneMetaStorageManager create(VaultManager vaultMgr, KeyValueStorage storage) {
+        return new StandaloneMetaStorageManager(
+                vaultMgr,
+                Mockito.mock(ClusterService.class),
+                Mockito.mock(ClusterManagementGroupManager.class),
+                Mockito.mock(LogicalTopologyService.class),
+                Mockito.mock(RaftManager.class),
+                storage
+        );
+    }
+
+    private final KeyValueStorage keyValueStorage;
+
+    /**
+     * The constructor.
+     *
+     * @param vaultMgr Vault manager.
+     * @param clusterService Cluster network service.
+     * @param cmgMgr Cluster management service Manager.
+     * @param logicalTopologyService Logical topology service.
+     * @param raftMgr Raft manager.
+     * @param storage Storage. This component owns this resource and will manage its lifecycle.
+     */
+    public StandaloneMetaStorageManager(VaultManager vaultMgr, ClusterService clusterService, ClusterManagementGroupManager cmgMgr,
+            LogicalTopologyService logicalTopologyService, RaftManager raftMgr, KeyValueStorage storage) {
+        super(vaultMgr, clusterService, cmgMgr, logicalTopologyService, raftMgr, storage);
+
+        keyValueStorage = storage;
+    }
+
+    @Override
+    public void start() {
+        keyValueStorage.start();
```

Comment:
You will also need to override the `stop` method in order to close the `keyValueStorage`